src/bench/addrman.cpp

```diff
@@ -112,10 +112,12 @@ static void AddrManGood(benchmark::Bench& bench)
      * we want to do the same amount of work in every loop iteration. */
 
     bench.epochs(5).epochIterations(1);
+    size_t addrman_count = bench.epochs() * bench.epochIterations();
 
-    std::vector<CAddrMan> addrmans(bench.epochs() * bench.epochIterations());
-    for (auto& addrman : addrmans) {
-        FillAddrMan(addrman);
+    std::vector<std::unique_ptr<CAddrMan>> addrmans(addrman_count);
+    for (size_t i{0}; i < addrman_count; i++) {
```

Comment:
579584d nit, prefer prefix iterator (idem in addrman_tests.cpp)
```suggestion
    for (size_t i{0}; i < addrman_count; ++i) {
```

Reply:
Fixed here. I don't think I've added any `i++` in addrman_tests.cpp.